Mapsui.Rendering.Skia-PCL/ArrowLineStringRenderer.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Mapsui.Geometries;
+using Mapsui.Providers;
+using Mapsui.Styles;
+using SkiaSharp;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Mapsui.Rendering.Skia
+{
+    public static class ArrowLineStringRenderer
+    {
+        public static void Draw(SKCanvas canvas, IViewport viewport, IStyle style, IFeature feature, IGeometry geometry, float opacity, SymbolCache symbolCache)
+        {
+            var arrowLineString = (ArrowLineString)geometry;
+
+            LineStringRenderer.Draw(canvas, viewport, style, feature, arrowLineString[0], opacity);
+
+            var arrowHead = arrowLineString.ArrowHead as Point;
+            var destination = viewport.WorldToScreen(arrowHead);
+            var deltaExtremity1 = new Point(arrowLineString.ArrowExtremities[0].X - arrowHead.X, arrowLineString.ArrowExtremities[0].Y - arrowHead.Y);
+            var deltaExtremity2 = new Point(arrowLineString.ArrowExtremities[1].X - arrowHead.X, arrowLineString.ArrowExtremities[1].Y - arrowHead.Y);
+            VectorStyle vectorStyle = new VectorStyle { Line = ((VectorStyle)style).Line };
```

Comment:
Why can't we directly use `var vectorStyle = style as VectorStyle`?

Reply:
On top of that: who calls `Draw`? Should we check if `IStyle` is actually a `VectorStyle`, or will that always be the case?